### PR TITLE
 Use lib wrapper for SUSEConnect 

### PR DESCRIPTION
**Pull request by rwx788**
Requires #6944

- [Verification run](http://f174.suse.de/tests/123#)

### CODE REVIEW

File: tests/x11/toolkits/prepare.pm [hunk 1]
```diff
@@ -28,11 +29,11 @@ sub run {
...
         }
         elsif (is_sle '>=15') {
-            assert_script_run 'SUSEConnect -p sle-module-development-tools/${VERSION_ID}/${CPU}';
+            add_suseconnect_product('sle-module-development-tools');
         }
         else {
             # for historical reasons we don't register SLE12 systems in openqa by default
-            assert_script_run 'SUSEConnect -r ' . get_required_var('SCC_REGCODE');
+            register_product();
             assert_script_run 'SUSEConnect -p sle-sdk/${VERSION_ID}/${CPU}';
```
Comment (foursixnine) [inline tests/x11/toolkits/prepare.pm]: @rwx788 shouldn't this line be
```add_suseconnect_product('sle-sdk',${VERSION_ID},${CPU})``

Comment (rwx788) [reply tests/x11/toolkits/prepare.pm]: Nice catch, yeah we can replace it with just 'add_suseconnect_product('sle-sdk') will update this PR.

**Comment by rwx788** [comment]
Hey @asdil12 @pdostal @lilyeyes, please, take a look. In case you think it might break anything, please, point me to some job I can clone to see that it works.
Thanks in advance!

**Comment by lilyeyes** [comment]
@rwx788 , I only checked "tests/security/selinux/selinux_smoke.pm", the changes looks good to me. 
Thanks for the changes.

**Comment by lilyeyes** [comment]
@rwx788  the o.s.d job: http://openqa.suse.de/tests/2517264

**Comment by asdil12** [comment]
Toolkits:
SLE15SP1: https://openqa.suse.de/t2520697
SLE12SP4: https://openqa.suse.de/t2520698

Machinery:
SLE15SP1: https://openqa.suse.de/t2520700
SLE12SP4: https://openqa.suse.de/t2520699

**Pull request merged**